pkgs/applications/radio/pat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pat";
+  version = "0.15.0";
+
+  src = fetchFromGitHub {
+    owner = "la5nta";
+    repo = "pat";
+    rev = "v${version}";
+    hash = "sha256-ydv7RQ6MJ+ifWr+babdsDRnaS7DSAU+jiFJkQszy/Ro=";
+  };
+
+  vendorHash = "sha256-TMi5l9qzhhtdJKMkKdy7kiEJJ5UPPJLkfholl+dm/78=";
+
```

Comment:
```suggestion

  ldflags = [ "-s" "-w" ];
```

Reply:
Tested working as expected. Updated in latest.

Reply:
waiting for ci...